src/plugins/vis_types/timelion/public/helpers/plugin_services.ts

```diff
@@ -6,12 +6,13 @@
  * Side Public License, v 1.
  */
 
-import type { IndexPatternsContract, ISearchStart } from 'src/plugins/data/public';
+import type { ISearchStart } from 'src/plugins/data/public';
+import type { DataViewsContract } from 'src/plugins/data_views/public';
 import type { ChartsPluginStart } from 'src/plugins/charts/public';
 import { createGetterSetter } from '../../../../kibana_utils/public';
 
 export const [getIndexPatterns, setIndexPatterns] =
```

Comment:
I hope we'll address these namings at some point too, but I understand it would be hard to manage in one PR 👌🏼

Reply:
Yes this is not the scope of this PR :)